JS-Week-3/NoteTakingApp.js

```diff
@@ -0,0 +1,86 @@
+const notes = [];
+
+function saveNote(content, id) {
+  const note = {
+    content,
+    id,
+  };
+  notes.push({ content, id });
```

Comment:
You've implemented both my comments here, but you've had to do only one or the other. Now you make define the `note` variable that you don't use anywhere.

Reply:
Hi @jason-vasilev ,Now  i have done the change , using note variable. Please check.